### PR TITLE
rework getting started intro a bit.

### PR DESCRIPTION
**Pull request by bkdotcom**
address issues raised in #23991
make it a bit more clear what javascript files are necessary and when..  move the "notes" to callouts

### CODE REVIEW

**Comment by bkdotcom** [comment]
additionally:  It would be nice show alternate usage of `bootstrap.bundle.min.js`  is it on cdn?
url and hash need added to _config.yml

**Comment by Johann-S** [comment]
currently `bootstrap.bundle.min.js` isn't on CDN because it's a new feature of our next release

File: docs/4.0/getting-started/introduction.md [hunk 1]
```diff
@@ -20,15 +20,26 @@ Copy-paste the stylesheet `<link>` into your `<head>` before all other styleshee
 <link rel="stylesheet" href="{{ site.cdn.css }}" integrity="{{ site.cdn.css_hash }}" crossorigin="anonymous">
 {% endhighlight %}
 
-Add our optional JavaScript plugins, jQuery, and [Popper.js](https://github.com/FezVrasta/popper.js) near the end of your pages, right before the closing `</body>` tag. Be sure to place jQuery and Popper.js first, as our JavaScript plugins depend on them. While we use [jQuery's slim build](https://blog.jquery.com/2016/06/09/jquery-3-0-final-released/) in our docs, the full version is also supported.
+
+#### Add our optional JavaScript components.
+
+If you wish to use our <a href="../../components/">javascript components</a>, you will need to place the following three scripts near the end of your pages, right before the closing `</body>` tag\*.   jQuery and Popper.js first, followed by bootstrap's script.
```
Comment (XhmikosR) [inline docs/4.0/getting-started/introduction.md]: Use Markdown syntax and remove multiple spaces

File: docs/4.0/getting-started/introduction.md [hunk 2]
```diff
@@ -20,15 +20,26 @@ Copy-paste the stylesheet `<link>` into your `<head>` before all other styleshee
...
 <script src="{{ site.cdn.jquery }}" integrity="{{ site.cdn.jquery_hash }}" crossorigin="anonymous"></script>
 <script src="{{ site.cdn.popper }}" integrity="{{ site.cdn.popper_hash }}" crossorigin="anonymous"></script>
 <script src="{{ site.cdn.js }}" integrity="{{ site.cdn.js_hash }}" crossorigin="anonymous"></script>
 {% endhighlight %}
 
-And that's it—you're on your way to a fully Bootstrapped site. If you're at all unsure about the general page structure, keep reading for an example page template.
+
+{% callout info %}
+* jQuery, and [Popper.js](https://github.com/FezVrasta/popper.js) are dependencies that must be included before our bootstrap script.
+* While we use [jQuery’s slim build](https://blog.jquery.com/2016/06/09/jquery-3-0-final-released/) in our docs, the full version is also supported.
```
Comment (XhmikosR) [inline docs/4.0/getting-started/introduction.md]: No need for you to use `’`; Jekyll does this for us.

File: docs/4.0/getting-started/introduction.md [hunk 3]
```diff
@@ -20,15 +20,26 @@ Copy-paste the stylesheet `<link>` into your `<head>` before all other styleshee
...
+* \* Scripts may also be placed before the closing `</head>` tag should your particulars require it.
+{% endcallout %}
+
+And that’s it—you’re on your way to a fully Bootstrapped site. If you’re at all unsure about the general page structure, keep reading for an example page template.
```
Comment (XhmikosR) [inline docs/4.0/getting-started/introduction.md]: Same here about `’`.

**Review by XhmikosR — CHANGES_REQUESTED** [review]

**Comment by mdo** [comment]
Closing for #24214.

**Pull request closed without merging**